src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -34,6 +37,22 @@ public QuestionDTO getQuestion(@PathVariable("id") Long id) {
         return questionConverter.convertToDTO(questionService.getById(id));
     }
 
+    @ApiOperation(value = "Get questions from the database by it's level id")
+    @GetMapping("/level/{levelId}")
```

Comment:
don't think frontend needs to know about level and module id. Let's just get the value and map it to corresponding enum